packages/api/.env.sample

```diff
@@ -4,7 +4,7 @@
 NODE_ENV=development
 PORT=3000
 # No ending slash
-BASE_URL=localhost:3000
+NEXT_PUBLIC_BASE_URL=localhost:3000
```

Comment:
In order to expose environment variable to the browser have you set this to NEXT_PUBLIC? @SpencerKaiser

Reply:
https://refine.dev/blog/next-js-environment-variables/#using-environment-variables-in-the-browser